### PR TITLE
typescript-promises  - Skylar Shaffer

### PR DESCRIPTION
**Pull request by skylarshaffer**
image

### CODE REVIEW

File: typescript-promises/main.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { takeAChance } from './take-a-chance.js';
+
+takeAChance('Skylar Shaffer')
+  .then((value) => console.log(value))
+  .catch((error) => console.log(error));
```
Comment (shawnkost) [inline typescript-promises/main.ts:5]: Rather than logging the whole `error` object, just log the `error.message`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change here Skylar!

File: typescript-promises/main.ts (hunk 1 above)
Comment (skylarshaffer) [reply typescript-promises/main.ts]: Fixed, thanks!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**